### ISSUE TITLE
Any game just crashes without useful logs

### ISSUE THREAD

**Issue by Fabi** [doitsujin/dxvk#1986]
I know Windows is not supported however about 2 years ago I remember using dxvk with some games and WildStar on Windows just fine.

Today I tried to use it again and not a single game will start. There is no crash log generated, nothing. Does not matter if dx9 or dx11 is used.

I just hope for some info on this issue. Currently I think it might have something to do with the Windows updates since them.
The only crash log that sometimes comes up is from the Event Viewer in the ntdll module inside the RtlGetFullPathName_Ustr but there is no more info.

### Software information
Windows 10 2004/2009 & Latest Insider

### System information
- GPU: AMD Vega 64
- Driver: Latest Stable
- DXVK version: 1.7.3/1.8.1

### Log files
Basically there is only one filled log (WildStar64_dxgi.log, the other ones are empty):

> info:  Game: WildStar64.exe
info:  DXVK: v1.8.1
info:  Found config file: dxvk.conf
info:  Effective configuration:
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
warn:  OpenVR: Failed to locate module

**Comment by Ryusennin** [comment]
Dxvk definitely works with Assassin's Creed 3, Origins and Odyssey on Windows 20H2 (just tested them again this weekend).

Make sure you're using both dxgi.dll and the appropriate d3dx.dll (32 or 64-bit depending on your game). I'm not familiar with WildStar and I don't know which DirectX version it uses. You might also want to refresh your dxvk.conf file just in case some options have changed.

**Comment by Fabi** [comment]
> Dxvk definitely works with Assassin's Creed 3, Origins and Odyssey on Windows 20H2 (just tested them again this weekend).
> 
> Make sure you're using both dxgi.dll and the appropriate d3dx.dll (32 or 64-bit depending on your game). I'm not familiar with WildStar and I don't know which DirectX version it uses. You might also want to refresh your dxvk.conf file just in case some options have changed.

Currently I use an empty config with default options and I use my 64 bit settings. I'm not sure if the errors is related to local loading of dll files in some applications. I have to go through https://docs.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-redirection to even make them loadable by any game.

This actually wasn't required back then.
And the logging just spams the output I already posted repeated in the log file. Nothing else happens. I tested it with a few friends on different machines too.

**Comment by Fabi** [comment]
ok I actually got the same issue here with 

> info:  Game: AC3SP.exe
info:  DXVK: v1.7.3
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
warn:  OpenVR: Failed to locate module

being spammed in AC3SP_dxgi.log

I wonder if that's an issue of the dynamic link library redirection since Windows blocks loading them nowadays by default on latest builds from the app directory. At least on fresh W10 installs for me

**Comment by K0bin** [comment]
DXVK on Windows is hit or miss because some games load either d3d11.dll or dxgi.dll (or both) from C:\Windows\System32. Some games work, some dont. Sometimes Vulkan drivers/loaders even load dxgi.dll which results in infinite recursion.
Nothing we can do about that.

**Comment by doitsujin** [comment]
Could also be some third-party software interfering. There's a reason why we don't really support Windows.

**Comment by Fabi** [comment]
Little update: This specific crash is caused by the dll redirection method that I linked. Patching the binary ends up with a different log that I'm trying to solve

**Comment by doitsujin** [comment]
The HUD itself has actually nothing to do with dxgi.dll, and DXVK simply wouldn't work if the wrong entry points were used.

How are you trying to enable the HUD?

**Comment by Fabi** [comment]
> The HUD itself has actually nothing to do with dxgi.dll, and DXVK simply wouldn't work if the wrong entry points were used.
> 
> How are you trying to enable the HUD?

I try it through the config file with the `DXVK_HUD=xxx` or through environment variables set in cmd. xxx of course replaced with multiple options from the readme. Nothing appears there tho.

**Comment by doitsujin** [comment]
the syntax for the config file is `dxvk.hud = xxx`.

> But yea the original issue is solved

I guess this can be closed then?

**Comment by Fabi** [comment]
> the syntax for the config file is `dxvk.hud = xxx`.
> 
> > But yea the original issue is solved
> 
> I guess this can be closed then?

ty!
